src/dbSta/src/dbNetwork.cc

```diff
@@ -4186,4 +4189,123 @@ Net* dbNetwork::getFlatNet(Net* net) const
   return dbToSta(db_net);
 }
 
+void dbNetwork::checkSanityNetConnectivity()
+{
+  // Check for hier net and flat net connectivity
+  dbSet<dbModNet> mod_nets = block()->getModNets();
+  for (auto mod_net : mod_nets) {
+    findRelatedDbNet(mod_net);
+  }
+
+  // Check for incomplete flat net connections
+  for (odb::dbNet* net : block_->getNets()) {
+    if (net->getITerms().size() + net->getBTerms().size() < 2) {
```

Comment:
An unused output would also fail this check but not really be an error (eg ff with Q and QN but only one is used).